docs/source/install.rst

```diff
@@ -40,3 +46,16 @@ Clone git repository and build maven project:
    $ git clone https://github.com/dask/knit
    $ cd knit
    $ python setup.py install mvn
+
+Testing on Docker
+~~~~~~~~~~~~~~~~~
+
+If you would like to test this package, but don't have a YARN cluster hanging around, you
+could make a small test one in your machine. This is essentially how the Continuous Integration tests
+work.
+
+   $ export CONTAINER_ID=`docker run -d mdurant/hadoop`
+   $ docker exec -it $CONTAINER_ID bash
+   # conda install dask distributed -y
+   # conda install -c conda-forge lxml py4j knit
```

Comment:
Maybe consolidate `dask` into this line?  We tend to keep conda-forge up-to-date.

Reply:
dask on conda-forge pulls in conda-forge definitions of bokeh and pandas, so you end up with matplotlib, qt and a load of other things. Perhaps it would be ok with dask-core.